.gitignore

```diff
@@ -2,6 +2,6 @@ protos/**/*.cache
 worlds/.*.wbproj
 build/
 
-controllers/NUgus_Controller/NUgus_Controller
+/controllers/NUgus_Controller/NUgus_Controller
 
-nugus_controller_test
+/nugus_controller_test
```

Comment:
Should this be a relative path?

Reply:
that file is in the root folder, so it shouldnt really matter